### PR TITLE
Add more movies to the list

### PR DESCRIPTION
**Pull request by aviskarkc10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +41,7 @@ A list of movies/series which could have really interesting porn parody names.
 | Lord of the Rings | Lord of the Rims |
 | Mission Impossible | Missionary Impossible |
 | Moon | Moan |
+| Mary Poppins | Cherry Poppins |
```
Comment (mesaugat) [inline README.md]: Sort position ⬆️

File: README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ A list of movies/series which could have really interesting porn parody names.
 | Fantastic Four | Fantastic Foursome |
 | Fight Club | Fist Club |
 | Forrest Gump | Forrest Hump |
+| Finding Dory | Binding Dory |
```
Comment (mesaugat) [inline README.md]: .

Comment (mesaugat) [reply README.md]: Sort 🆙

Comment (aviskarkc10) [reply README.md]: done isn't it?

Comment (mesaugat) [reply README.md]: Forrest Gump comes before Finding Dory?

Comment (aviskarkc10) [reply README.md]: Am I too sleepy or what? it's okay here
image

Comment (mesaugat) [reply README.md]: Oh my! Seems like a GitHub issue. 

Looks good now.

Comment (mesaugat) [reply README.md]: ¯\\_(ツ)_/¯

Screen Shot 2019-07-30 at 10 03 12 PM

**Pull request merged**